src/commonMain/kotlin/exchange.dydx.abacus/state/v2/supervisor/SubaccountSupervisor.kt

```diff
@@ -698,7 +700,7 @@ internal class SubaccountSupervisor(
                 transferPayloadString,
                 null,
                 isolatedMarginTransactionCallback,
-                useTransactionQueue,
+                true,
```

Comment:
let's use named params for booleans

Reply:
We should try to use named params for all calls if possible.  

Android Studio will do this for you.  Click on the light-bulb will give you the option:

<img width="476" alt="Screenshot 2024-06-06 at 18 46 12" src="https://github.com/dydxprotocol/v4-abacus/assets/102453770/59e27738-e630-468a-8c77-cd3f4a0871eb">